app/Policies/GameSetPolicy.php

```diff
@@ -37,13 +37,11 @@ public function view(?User $user, GameSet $gameSet): bool
 
     public function create(User $user): bool
     {
-        return false;
-
-        // TODO enable after dropping GameAlternatives
-        // return $user->hasAnyRole([
-        //     Role::ADMINISTRATOR,
-        //     Role::GAME_EDITOR,
-        // ]);
+        return $user->hasAnyRole([
+            Role::ADMINISTRATOR,
+            Role::GAME_HASH_MANAGER,
+            Role::GAME_EDITOR,
+        ]);
```

Comment:
I'm not sure if we should have a separate role, or if this should be opened more.

I know we have hubs dedicated to events, and there's all the `Meta|**` hubs that are used by other teams.

Reply:
I'm not too sure either. I don't think this should be opened up to events managers or the 3 staff dev roles. Right now I'm inclined to think if any of those folks regularly find themselves creating new hubs, they should be given the game editor role, otherwise they can ask one of these 3 roles who are ultimately responsible for the curation of the hubs.